docs/design/proposals/carm/cross-account-resource-management.md

```diff
@@ -20,28 +20,36 @@ The proposed solution will solve the problem of cross account management within
 
 Controllers will store AccountIDs and their associated IAM Role ARNs in a `ConfigMap` Kubernetes object. Cluster administrators will be responsible for administrating and updating the `ConfigMap`, and creating the target roles using the AWS Console/CLI. Controllers can reuse and existing IAM Role ARN if it matches the scope they need.
 
-To prevent unauthorized users from making any CRUD operations on resources in different AWS accounts, ACK will associate a unique Namespace to each AccountID, which will help administrators limit access to namespaces using K8s RBAC model, hence limit access to AccountIDs.
+To prevent unauthorized users from making any CRUD operations on resources in different AWS accounts, cluster administrators will be able to set an AWS Owner Account ID as an annotation on each namespaces, which will be used by ACK Controllers to override the AWS Account ID that is used the create the resources in that namespace. This will help administrators limit access to namespaces using K8s RBAC model, hence limit access to AccountIDs.
```

Comment:
AWS Owner Account ID as an annotation on each namespaces -> **namespace** 
override the AWS Account ID that is used the create  -> that is used **to** create